data/tutorials/guides/1wf_06_database.md

```diff
@@ -0,0 +1,328 @@
+---
+id: database
+title: Database handling
+description: >
+  A guide to database handling in OCaml with the Caqti and ppx_rapper libraries
+category: "Guides"
+---
+
+## Introduction
+
+The OCaml ecosystem provides multiple libraries which can be used to access a RDBMS.
+
+One of these libraries is `caqti` which has multiple advantages:
+
+- it can be used with multiple RDBMS (Postgres, MariaDB, SQLite3),
+- it can be used in a Lwt context which permits its use in some application server. Especially
+the Dream framework which has some specific functions which helps using Caqti,
+- it can be used with the `ppx_rapper` preprocessor. This preprocessor converts an annoted
+SQL query into a query function which leverage the strong typing feature of OCaml.
```

Comment:
```suggestion
- It can be used with multiple RDBMS (Postgres, MariaDB, SQLite3).
- It can be used in a Lwt context. That permits use in some application servers. Especially the Dream framework, that has some dedicated functions that help using Caqti.
- It can be used with the `ppx_rapper` preprocessor. This preprocessor converts an annotated SQL query into a query function that leverages the strong typing feature of OCaml.
```